README.md

```diff
@@ -47,13 +51,18 @@ See aces/Loris README.md for further information and Loris installation informat
  * What prod file name would you like to use? default: prod  [leave blank]
  * Enter the list of Site names (space separated) site1 site2
 
-Ensure that /home/$lorisadmin/.bashrc includes the statements: 
+  If the imaging install script reports errors in creating directories (due to /data/ mount permissions), manually execute mkdir and chmod commands starting at [imaging_install.sh:L90](https://github.com/aces/Loris-MRI/blob/master/imaging_install.sh#L90)
+
+5. Configure paths and environment
+
+   Ensure that /home/$lorisadmin/.bashrc includes the statement: 
+
+   ```source /data/$projectname/bin/mri/environment```
+
+   Ensure that the apache envvars file includes all the EXPORT statements from minc-toolkit-config.sh (this file located in the path where the MINC tools are installed). This is necessary for BrainBrowser to successfully load MINC images.
 
```

Comment:
@christinerogers,

this is no longer necessary due to the latest changes in this pull request:
https://github.com/aces/Loris/pull/1843
But instead, we can specify that the config.xml file (on the Loris side) should include, as documented in the above pull request:
  <!-- MINC TOOLS PATH -->
    <MINCToolsPath>/opt/minc/</MINCToolsPath>
(where /opt/minc/ is the MINC tools path in this example).